### ISSUE TITLE
Is there a way to get the error response in JSON?

### ISSUE THREAD

**Issue by team-hq** [TheAlphaSchoolSystemPTYLTD/api-introduction#12]
Right now the, when I get a 403 or 501, the error response is in HTML and the JSON response in appended to the end of the HTML. Is there a way to a clean JSON response? I have 'content-type': 'application/json' inside my header. Is there something else one needs to do? 

This is the current response and it's hard to parse:

`'<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">\r\n<html xmlns="http://www.w3.org/1999/xhtml">\r\n<head>\r\n<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1"/>\r\n<title>403 - Forbidden: Access is denied.</title>\r\n<style type="text/css">\r\n\r\n</style>\r\n</head>\r\n<body>\r\n<div id="header"><h1>Server Error</h1></div>\r\n<div id="content">\r\n <div class="content-container"><fieldset>\r\n  <h2>403 - Forbidden: Access is denied.</h2>\r\n  <h3>You do not have permission to view this directory or page using the credentials that you supplied.</h3>\r\n </fieldset></div>\r\n</div>\r\n</body>\r\n</html>\r\n{"__status":"invalid","__msg":"A Security Role must be defined in the TASS.web API configuration for Version 3 and above.","__invalid":{}}'
`

**Comment by scottetass** [comment]
JSON is getting returned when an error is raised now.

![image](https://github.com/TheAlphaSchoolSystemPTYLTD/api-introduction/assets/149555853/3cd6c91b-6f46-4df7-9eb6-11f225fdfbfb)

As this is a very old issue, I am going to close it. If the problem still persists, please re-open this item.
